tableprint/printer.py

```diff
@@ -18,12 +18,12 @@
 from six import string_types
 
 from .style import LineStyle, STYLES
-from .utils import ansi_len, format_line, parse_width
+from .utils import ansi_len, format_line, parse_width, calculate_widths
 
 __all__ = ('table', 'header', 'row', 'hrule', 'top', 'bottom', 'banner', 'dataframe', 'TableContext')
 
 STYLE = 'round'
-WIDTH = 11
```

Comment:
I think we should switch from a global default to function specific defaults (see below)